py/tests/unit/with_runtime_sparkling/test_drf.py

```diff
@@ -0,0 +1,115 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+from pyspark.ml import Pipeline, PipelineModel
+from pyspark.mllib.linalg import *
+from pyspark.mllib.linalg import *
+from pyspark.sql.types import *
+from pyspark.sql.types import *
+from pysparkling.ml import H2ODRF
+
+def testParams():
+    drf = H2ODRF(binomialDoubleTrees=False,
```

Comment:
What is the purpose of this test? Wouldn't it be better to set values to something non-default?

Reply:
Initially, the purpose of this test was that all options can be passed to the algorithms without any exceptions - back then we did not have the type converters validating the input.

Maybe we do not need to do such tests anymore and just create separate test suite for the type converts. WDYT?

Reply:
I'm definitely for having such a tests in Python. Typo can be made very easily. Just would like to understand the context. Just wondering whether it's possible to write such a test in a generic way via reflection to able to test all algorithms with one logic.

Reply:
I will do this in a following PR what do you think? Already started working on it

Reply:
ok, no problem